code-challenges.js

```diff
@@ -1,64 +1,117 @@
 // ASSESSMENT 2: Coding Practical Questions
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-// Use the test variables provided.
-
-var num1 = 15
-// Expected output: "15 is divisible by three"
-
-var num2 = 0
-// Expected output: "0 is divisible by three"
-
-var num3 = -7
-// Expected output: "-7 is not divisble by three"
-
-
-
-
+// // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// // Use the test variables provided.
+
+// var num1 = 15;
+// // Expected output: "15 is divisible by three"
+// var num2 = 0;
+// // Expected output: "0 is divisible by three"
+// var num3 = -7;
+// // Expected output: "-7 is not divisble by three"
+
+
+// // i got the concept. but the problem is that it only checks for the first argument and then goes answers num1 and never checks for num2. it is 3:28 pm on sunday. i will go on and then come back later to that problem :) time spent: 4,5 hrs
+
+
+// function divisibleNum(num) {
+//     if (num % 3 === 0){
+//         return `${num1} is divisible by three`;
+//     } if  (num === 0 ){
+//         return `${num2} is divisible by three`;
+//     } else if (num % 3 < 0 ){
+//         return `${num3} is not divisible by three`;
+//     }   else {
+//         return "error";
+//     }
+    
+        
+// }
+//     console.log (divisibleNum(num1));
+//     console.log (divisibleNum(num2));
+//     console.log (divisibleNum(num3));
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
-var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
-
+// THIS IS NOT WORKING I COULNDT FIGURE IT OUT :(
 
 
+// var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"];
+// // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+//     // turn an array into string
+// var randomNounsStr = randomNouns.join(', ');
+//     // give a string a function
 
-// --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
+// for (let i = 0; i < randomNounsStr.length; i++) {
+//     if (typeof randomNounsStr[i] === 'string') {
+//         randomNounsStr[i] = randomNounsStr[i].toUpperCase();
+//     }
+// console.log(randomNounsStr());
```

Comment:
this solution is suuuper close! you're on the right track. when we do review of the assessment it'll help.